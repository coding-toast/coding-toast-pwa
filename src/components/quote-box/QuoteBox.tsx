import * as React from 'react';
import { Jumbotron } from 'react-bootstrap';

const QuoteBox: React.FC = () => (
  <Jumbotron className='quote-box'>
    <blockquote className={'text-color-primary text-style-lead text-quote'}>
      Because it takes more than going over your git log before standup to become a great engineer.
    </blockquote>
  </Jumbotron>
);

export default QuoteBox;
