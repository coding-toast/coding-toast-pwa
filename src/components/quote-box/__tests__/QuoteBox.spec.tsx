import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import QuoteBox from '../QuoteBox'

describe('<QuoteBox />', () => {
  it('renders correctly', () => {
    const component = shallow(<QuoteBox />)
    expect(toJson(component)).toMatchSnapshot()
  })
})
