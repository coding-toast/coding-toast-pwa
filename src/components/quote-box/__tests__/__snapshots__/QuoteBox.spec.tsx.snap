// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QuoteBox /> renders correctly 1`] = `
<Jumbotron
  className="quote-box"
  fluid={false}
>
  <blockquote
    className="text-color-primary text-style-lead text-quote"
  >
    Because it takes more than going over your git log before standup to become a great engineer.
  </blockquote>
</Jumbotron>
`;
